src/test/integration/things-test.js

```diff
@@ -148,6 +148,37 @@ describe('things/', function() {
     expect(err.response.status).toEqual(404);
   });
 
+  it('rename a thing', async () => {
```

Comment:
It would be nice to also test failing to rename a thing (i.e. missing name and optionally name = '')